core/src/main/java/bisq/core/locale/CurrencyUtil.java

```diff
@@ -267,32 +267,42 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         return currencies;
     }
 
-    //https://www.revolut.com/pa/faq#can-i-hold-multiple-currencies
+    // https://www.revolut.com/help/getting-started/exchanging-currencies/what-fiat-currencies-are-supported-for-holding-and-exchange
     public static List<TradeCurrency> getAllRevolutCurrencies() {
         ArrayList<TradeCurrency> currencies = new ArrayList<>(Arrays.asList(
-                new FiatCurrency("USD"),
-                new FiatCurrency("GBP"),
-                new FiatCurrency("EUR"),
-                new FiatCurrency("PLN"),
+                new FiatCurrency("AED"),
+                new FiatCurrency("AUD"),
+                new FiatCurrency("BGN"),
+                new FiatCurrency("CAD"),
                 new FiatCurrency("CHF"),
+                new FiatCurrency("CZK"),
                 new FiatCurrency("DKK"),
-                new FiatCurrency("NOK"),
-                new FiatCurrency("SEK"),
-                new FiatCurrency("RON"),
-                new FiatCurrency("SGD"),
+                new FiatCurrency("EUR"),
+                new FiatCurrency("GBP"),
                 new FiatCurrency("HKD"),
-                new FiatCurrency("AUD"),
-                new FiatCurrency("NZD"),
-                new FiatCurrency("TRY"),
-                new FiatCurrency("ILS"),
-                new FiatCurrency("AED"),
-                new FiatCurrency("CAD"),
+                new FiatCurrency("HRK"),
                 new FiatCurrency("HUF"),
+                new FiatCurrency("HUF"),
+                new FiatCurrency("ILS"),
                 new FiatCurrency("INR"),
```

Comment:
I don't see `INR` in their supported currencies.

Reply:
That's true! Nice catch, I unified the previous list with current list so it stayed there.

It appears Revolut discontinued INR support (https://community.revolut.com/t/revolut-in-india-not-getting-interbank-rates/16999/14).